rustfmt-core/rustfmt-lib/tests/source/issue-4020.rs

```diff
@@ -0,0 +1,9 @@
+// rustfmt-wrap_comments: true
+
```

Comment:
I don't think it's mentioned in the Contributing docs, but for awareness, when the test is just ensuring rustfmt doesn't modify the formatting like this one you only need the test file under `target` and can skip the one under `source`

Reply:
Thanks for that! Removed now.